microsoft-edge/web-platform/ad-selection-api.md

```diff
@@ -0,0 +1,297 @@
+
+Each buyer or seller can deploy multiple instances of a given service, such as:
+* **SellerFrontEnd** and **Auction**.
+* **BuyerFrontend**, **Bidding**, and **K-Anonymity**.
+
+Some of these services take an input which is a JavaScript worklet that defines User-Defined Functions.
+
+
+<!-- ---------- -->
+###### User-Defined Functions for sellers
+
+| UDF | Service in which the UDF runs | Description |
+| --- | --- | --- |
+| `scoreAd()` | **Auction** | Generates a score for each buyer's bid, or rejects the bid.  This score is then used by the **SellerFrontEnd** service, to choose a winner from among all buyers. |
+| `reportResult()` | **SellerFrontEnd** | For event-level reporting.  Runs in the **SellerFrontEnd** service after the final winner has been chosen.  Notifies the seller about the winning bidder, and provides the bid value. |
+
+
+<!-- ---------- -->
+###### User-Defined Functions for buyers
+
+| UDF | Service in which the UDF runs | Description |
+| --- | --- | --- |
+| `generateBids()` | **Bidding** | Generates a bid, and chooses the banner that will be shown. |
+| `reportWin()` | **SellerFrontEnd** | For event-level reporting.  Runs in the **SellerFrontEnd** service, if the buyer's bid wins the auction.  Notifies the buyer that they have won the auction, and generates notification URLs that will be triggered later, such as during banner rendering or other client events. |
+| `getValues()` | **Bidding Selection & Key/Value** | Runs in the **Bidding Selection & Key/Value** service, if the buyer uses that service.  Allows for custom code execution for data lookup and selection. |
+
+
+<!-- ------------------------------ -->
+#### Deployment guide
+
+To deploy the Ad Selection API service to Azure:
+
+1. Create an Azure account and create a subscription at [Microsoft Azure](https://azure.microsoft.com).
+
+1. Choose the **Confidential ACO containers** option.
+
+1. Download the Terraform [deployment scripts and guide](https://go.microsoft.com/fwlink/?linkid=2290115).  **Note:** Make sure the hash of the image you downloaded matches the hash in the script you download.
+
+1. The Terraform scripts already include links to public images that are provided by Microsoft.  Choose the required services and deploy them by following the steps in the included `readme.md`.
+
+   1. Set any specific flags for your environment, including auction worklet locations.
+
+   1. Configure all prerequisites and prepare all tools for deployment.
+
+   1. Create Azure Resources and run the Terraform script that finishes deployment.
+
+1. Develop worklets including `user-defined-function` (UDF) and start Ad Selection API services.
+
+
+<!-- ====================================================================== -->
+## Use the Ad Selection API on your website
+
+To start using the Ad Selection API, sellers and buyers need to use the API, by modifying the JavaScript code of their websites running in Microsoft Edge.
```

Comment:
unclear at "modify the JavaScript code of their websites running in Microsoft Edge."  can we add words to clarify?  

"browser detection;  add code to detect if the user's browser is Edge, if so, run certain JS"
@yanushP

Reply:
closing; Outdated markdown; fresh [comment](https://github.com/MicrosoftDocs/edge-developer/pull/3285/files#r1790831659)